cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java

```diff
@@ -88,6 +88,13 @@ public void clearInternalData(){
         editor.commit();
     }
 
+    public HashMap getTCData(){
+        HashMap tcData = new HashMap();
+        for(String key : pref.getAll().keySet()) if (key.startsWith(IABTCF_KEY_PREFIX))
+            tcData.put(key, pref.getString(key, null));
+        return tcData;
+    };
+
     public void clearConsentData(){
         for(String key : pref.getAll().keySet()) if (key.startsWith(IABTCF_KEY_PREFIX))
```

Comment:
I think you can reuse the method you implemented above `getTCData()` in the `clearConsentData`.